onebusaway-android/src/main/AndroidManifest.xml

```diff
@@ -268,6 +276,16 @@
                 <action android:name=".nav.NavigationReceiver" />
             </intent-filter>
         </receiver>
+        <receiver
+            android:name=".nav.FeedbackReceiver"
+            android:enabled="true"
+            android:exported="true" >
+            <intent-filter>
+                <action android:name=".nav.FeedbackReceiver" />
+                <action android:name="${applicationId}.action.SCHEDULE"/>
```

Comment:
`<action android:name="${applicationId}.action.SCHEDULE"/>` and `<action android:name="${applicationId}.action.POLL"/>` should be removed here and replaced with the Intents that the FeedbackReceiver actually receives, which should be in the same format of:

`<action android:name="${applicationId}.action.REPLY"/>`

Reply:
You'll also need your delete intent notification for when the feedback notification gets dismissed listed here:
<action android:name="${applicationId}.action.DISMISS_FEEDBACK"/>

Reply:
Upon further discussion this isn't needed unless we expose the receiver to external apps, so let's just remove the `<intent-filter>` block

Reply:
We have removed the `<intent-filter>` block from AndroidManifest as we don't need to expose the receiver to other apps.